### DISCUSSION TITLE
How to hide "Feature activated" info text during boot?

### DISCUSSION THREAD

**Discussion by yzkzero** [gopro/labs#255] [Q&A] [answered]
I found the "Feature activated: xxx" info text displays a few seconds longer than I set in DSPL=x. But the owner info text duration is correct or at least like that value. 
My question is if there's a way to turn off this message?

**Comment by dnewman-gpsw** [comment, marked as answer]
The more more features activated to longer the message shows (more to read/remind.)  It is there for safety and tech support reasons, so there isn't a away to completely remove it.  !MDSPL=10 is the only way to much is much less.

**Reply by yzkzero** [reply]
Thanks for the answer. 
But setting DSPL=10 also makes owner info text too fast to read...
